app/styles/components/_users-pagination-links.scss

```diff
@@ -0,0 +1,14 @@
+.users-pagination {
```

Comment:
file name should be user-pagination of else the class name should be user-pagination-links